src/test/scala/io/iohk/ethereum/vm/MockWorldState.scala

```diff
@@ -12,7 +12,8 @@ object MockWorldState {
 case class MockWorldState(
   accounts: Map[Address, Account] = Map(),
   codeRepo: Map[Address, ByteString] = Map(),
-  storages: Map[Address, MockStorage] = Map()
+  storages: Map[Address, MockStorage] = Map(),
+  headers: Map[BigInt, ByteString] = Map()
```

Comment:
But why? It's not the world's responsibility to limit those hashes, it should be consequence of the opcode implementation. And in fact you cannot test it properly if the world is already limiting them.